test/built-ins/Temporal/Duration/prototype/round/balances-up-to-weeks.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018 Bloomberg LP. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.duration.prototype.round
+description: Balances up to weeks correctly when years and months are present.
```

Comment:
Nitpick about the terminology:
```suggestion
description: Rounds up to weeks correctly when years and months are present.
```
I'd suggest renaming the test file as well.

(Rounding is when the duration is modified, balancing is when the duration remains the same length but is expressed in different units)